src/test/java/org/reaktivity/specification/socks/internal/SocksFunctionsTest.java

```diff
@@ -44,43 +48,136 @@ public void setUp() throws Exception
     }
 
     @Test
-    public void shouldLoadFunctions() throws Exception
+    public void shouldLoadFunctionsWithDomainName() throws Exception
```

Comment:
Please revert test name change back to `shouldLoadFunctions`.

We just want to load the functions to make sure they are registered properly, any function will do here.

Reply:
done